src/styles/reset.less

```diff
@@ -11,13 +13,99 @@
  * make sure we're resetting all spacing elements.
  *
  */
+
+  // Light Theme Variables
+ .light-theme {
+  --theme: light;
```

Comment:
- would be useful to have css variable colors be prefixed by `color` accordingly, e.g. `theme-color-primary-100`
- use more consistent variable naming (e.g. `--[...]surface-1` -> `--[....]-surface-primary` - reserve numbers at the end for lightness or w/e)
- in general i dont really like some of the variable naming lol(eg `-text-on-background-1` doesn't really tell much - ideally, for a variable like that we shouldn't need a comment by its declaration saying what it's for, the variable name should describe itself)

Reply:
I agree with the point about prefixing color, but in terms of variable naming, that's a reflection of how the existing variables are named, and isn't something that needs to be cleaned up in this PR. I added #540.

Reply:
Generally, I am a proponent of `color-weight` for variable names, and then adding aliases of primary and secondary default colors than link to said colors.